### PR TITLE
Added note about ISP blocking port 445

### PR DESCRIPTION
**Pull request by rogrange**

### CODE REVIEW

File: downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc [hunk 1]
```diff
@@ -19,6 +19,8 @@ Once the installation completes, these are the default protocol and ports used:
 
 These can be changed. Consult the *README.md* for further details. It is recommended that you leave the defaults unless you need to change them due to port conflicts or other factors.
 
+NOTE: Port 445 has been known to be blocked by some ISPs. If port 445 is blocked please refer to the README.md file for more information on changing port values.
```
Comment (sayjadha) [inline downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc:22]: Suggest a comma after blocked -> '...blocked,'

Comment (sayjadha) [inline downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc:22]: Should this be in monospace text as it's a file name?

**Review by sayjadha — COMMENTED** [review]
A suggestion and a question, rest LGTM.

File: downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc (hunk 1 above)
Comment (rogrange) [reply downstream/modules/platform/ref-accessing-control-auto-hub-eda-control.adoc]: I agree. Done.

**Comment by sayjadha** [comment]
Changes LGTM.

**Pull request merged**